TAF_EasyRest/TestFramework/Tools/DBConnectionWrapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Npgsql;
+
+namespace TestFramework.Tools
+{
+    internal class DBConnectionWrapper
+    {
+
+        private static readonly string pg_connectionString = "Host=localhost;Username=admin;Password=12345678;Database=easyrest";
+
+        static void ShowHeaders(NpgsqlDataReader reader)
+        {
+            var columnNames = Enumerable.Range(0, reader.FieldCount)
+                        .Select(reader.GetName)
+                        .ToArray();
+            foreach (var column in columnNames) { Console.Write($"{column}\t"); }
+            Console.WriteLine();
+        }
+
+        static void ShowRows(NpgsqlDataReader reader)
+        {
+            while (reader.Read())
+            {
+                var row = Enumerable.Range(0, reader.FieldCount)
+                    .Select(reader.GetValue).ToArray();
+                foreach (var column in row) { Console.Write($"{column}\t"); }
+                Console.WriteLine();
```

Comment:
why do you need this?

Reply:
I need this to make a new line for each row of the table, because I use _while (reader.Read())_ loop - that means to show many rows